src/client_types.ts

```diff
@@ -130,3 +134,8 @@ export type NodeGetHistoryParams = latest.NodeGetHistoryParams
 export type NodeGetHistoryResult = latest.NodeGetHistoryResult
 
 export type NodeDeleteHistoryParams = latest.NodeDeleteHistoryParams
+
+export type NodeSubscribeParams = cV33.NodeSubscribeParams;
```

Comment:
please use "latest" everywhere. So it can be changed later afterwards.

Reply:
done